docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/TimeSeries/DetectIidChangePoint.cs

```diff
@@ -30,7 +30,7 @@ public IidChangePointData(float value)
 
         // This example creates a time series (list of Data with the i-th element corresponding to the i-th time slot). 
         // IidChangePointDetector is applied then to identify points where data distribution changed.
```

Comment:
>IidChangePointDetector [](start = 11, length = 22)

let's not put class names in the comments. This has already changed and is incorrect now. we can't catch these.